packages/ui/components/DropdownMenu.tsx

```diff
@@ -18,6 +18,7 @@ interface DropdownMenuProps {
   items: DropdownMenuItem[]
   side?: 'top' | 'right' | 'bottom' | 'left'
   align?: 'start' | 'center' | 'end'
+  onSelect?: (options: string[]) => void
```

Comment:
```suggestion
```

Reply:
@KhudaDad414 onSelect prop is being used at PropertyControl.tsx and at VisualEditor.tsx